EIPS/eip-7833.md

```diff
@@ -0,0 +1,474 @@
+---
+eip: 7833
```

Comment:
```suggestion
eip: 7533
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
OK, done, thx